src/main/java/com/onarandombox/multiverseinventories/util/CommentedYamlConfiguration.java

```diff
@@ -128,8 +123,8 @@ public boolean save() {
                         }
                         // Find out if the current depth (whitespace * 2) is greater/lesser/equal to the previous depth
                         if (whiteSpace / 2 > depth) {
```

Comment:
Looks like `whiteSpace / 2` done a vary places, may be a good idea to do it once and store it into a variable for reuse.

Reply:
Yup, good idea.